packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -122,11 +122,12 @@ module.exports = {
     loaders: [
       // Default loader: load all assets that are not handled
       // by other loaders with the url loader.
-      // Note: This list needs to be updated with every change of extensions
+      //
+      // Note:
+      // The "url" loader exclude list needs to be updated with every change of extensions
       // the other loaders match.
-      // E.g., when adding a loader for a new supported file extension,
-      // we need to add the supported extension to this loader too.
-      // Add one new line in `exclude` for each loader.
+      // E.g., when adding a loader for a new supported file extension (such as .sass),
```

Comment:
Let's say "For example" rather than "E.g.", as it's not a very good way to start a sentence.